tests/testthat/test-int-latent_individual.R

```diff
@@ -1,3 +1,9 @@
+stochastic_test_message <- paste0(
```

Comment:
I think as this is developer side I would be tempted by just adding a comment to this effect vs printing it into the test message?

Reply:
Yeah fair, just a comment at the top of the script?

Reply:
Printing it to the test message makes it clear which tests this applies to

Reply:
I could also put the comment inside each of the relevant tests

Reply:
> I could also put the comment inside each of the relevant tests

So I think this is what I would do but I don't feel that strongly so your call on what you think is clearer/most useful

Reply:
I don't have strong preferences either. I slightly dislike copy pasting the same message, so perhaps put it at the top and then a note to refer to the comment in each function

Reply:
I've implemented that approach in c7638e5